### ISSUE TITLE
training config used for training stt_en_quartznet15x5

### ISSUE THREAD

**Issue by ROZBEH** [NVIDIA/NeMo#9140]
This is not a feature request per se. It's more about seeking help :-)

Right now I am trying the `stt_en_quartznet15x5` in nemo and I can load it and run inference without any issues. I can also see the model config which includes the layers, post and preprocessing parts without any issue.

Now, I am trying to train such a model from scratch. I used the default config that comes with the nemo repo like the ones located [here](https://github.com/NVIDIA/NeMo/tree/main/examples/asr/conf/quartznet). However, this won't produce a model with a performance on par with the model that ships with nemo(`stt_en_quartznet15x5`). 

While I can see and find the model configs located [here](https://github.com/NVIDIA/NeMo/tree/main/examples/asr/conf/quartznet), I was wondering if there is a way to see the actual config used by NVIDIA to train the aforementioned model. Things like training configs, batches, epochs, learning rate, etc. that might have been changed in the updated config.

Thank you.

**Comment by nithinraok** [comment]
@ROZBEH generally we try to put config to suit for training on small number of GPUs but for training these models we might have used large number of GPUs, so parameters might not be the same. 

@sam1373 do you know the difference between current config and model trained?

**Comment by ROZBEH** [comment]
I see - thanks @nithinraok 
Yeah knowing the actual config can be really beneficial.
